zebedee-cms/src/main/java/com/github/onsdigital/zebedee/user/service/UsersServiceImpl.java

```diff
@@ -308,15 +285,31 @@ public User removeKeyFromKeyring(String email, String keyIdentifier) throws IOEx
 
     @Override
     public User update(Session session, User user, User updatedUser) throws IOException, UnauthorizedException,
-            NotFoundException, BadRequestException {
+            NotFoundException, BadRequestException, EmailException {
+
         if (!permissionsService.isAdministrator(session.getEmail())) {
             throw new UnauthorizedException("Administrator permissionsServiceImpl required");
         }
 
         if (!userStore.exists(user.getEmail())) {
             throw new NotFoundException("User " + user.getEmail() + " could not be found");
         }
-        return update(user, updatedUser, session.getEmail());
+
+        // TODO bcrypt this code
+        String code = UUID.randomUUID().toString();
+
+        if (updatedUser.getOwnerEmail() != user.getOwnerEmail()) {
```

Comment:
might be a bit beyond the scope of this but if a password reset is enforced do we also want to find and kill the user session?